ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py

```diff
@@ -5,6 +5,8 @@
 # local
 import ivy_tests.test_ivy.helpers as helpers
 import ivy.functional.backends.tensorflow as ivy_tf
+import ivy.functional.backends.numpy as ivy_np
```

Comment:
We should be using `ivy_tf` for generating data for tensorflow frontend tests

Reply:
done